modules/mauth-proxy/src/main/java/com/mdsol/mauth/proxy/ProxyConfig.java

```diff
@@ -54,8 +59,28 @@ public String getPrivateKey() {
   public UUID getAppUuid() {
     return appUuid;
   }
-  public boolean isV2OnlySignRequests() {
-    return v2OnlySignRequests;
+
+  public List<MAuthVersion> getSignVersions() { return signVersions; }
+
+  public List<MAuthVersion> getSignVersions(String signVersionsStr) {
+    List<MAuthVersion> signVersions = new ArrayList();
+    List<String> versionList = Arrays.asList(signVersionsStr.toUpperCase().split(","));
+    versionList.forEach(e -> {
+      switch (e.toLowerCase()) {
+        case "v1":
+          signVersions.add(MAuthVersion.MWS);
+          break;
+        case "v2":
+          signVersions.add(MAuthVersion.MWSV2);
+          break;
```

Comment:
Can we log a warning if an unrecognized string is passed in?

Reply:
All these comments apply to the DefaultSIgner too

Reply:
updated as suggested